### ISSUE TITLE
Remove need for enum34 for Python 3

### ISSUE THREAD

**Issue by saparikh** [intentionet/netconan#122]
`enum34` Python module is not compatible with certain Ansible modules that require the `enum` module. Since `enum34` is not required for Python 3, we need to adjust the inclusion/usage to just Python 2 deployments.

**Comment by sfraint** [comment]
What steps are needed to reproduce the issue @saparikh ?
The three notebooks [here](https://github.com/saparikh/bf-ansible-tower-demo) run without errors on my setup which has `enum34` installed.

I ran the notebooks as follows:
```
ansible-playbook -i inventory --extra-vars "batfish_server=localhost candidate_snapshot=candidate_snapshot bf_network=bf_network hostname=lhr-leaf-03 snapshot_dir=path/example" create_candidate_snapshot.yml
```

```
ansible-playbook -i inventory --extra-vars "demo_base_dir=path/bf-ansible-tower-demo/ demo_snapshot_dir=path/bf-ansible-tower-demo/snapshots/ tmp_cfg_dir=path/bf-ansible-tower-demo/tmp/ hostname=lhr-leaf-03 BGP_AS=12345" generate_router_config.yml
```

```
ansible-playbook -i inventory --extra-vars "batfish_server=localhost candidate_snapshot=candidate_snapshot bf_network=bf_network hostname=lhr-leaf-03 snapshot_dir=path/example" validate_configuration_change.yml
```

My setup is:
```
>>> python --version
Python 3.7.2
```

```
>>> pip freeze
ansible==2.8.2
asn1crypto==0.24.0
attrs==19.1.0
bidict==0.18.0
certifi==2019.6.16
cffi==1.12.3
chardet==3.0.4
ConfigArgParse==0.14.0
cryptography==2.7
deepdiff==3.3.0
Deprecated==1.2.6
enum34==1.1.6
idna==2.8
ipaddress==1.0.22
Jinja2==2.10.1
jsonpickle==1.2
MarkupSafe==1.1.1
netconan==0.10.1
numpy==1.16.4
pandas==0.24.2
passlib==1.7.1
pybatfish==0.36.0
pycparser==2.19
python-dateutil==2.8.0
pytz==2019.1
PyYAML==5.1.1
regex==2019.6.8
requests==2.22.0
requests-toolbelt==0.9.1
simplejson==3.16.0
six==1.12.0
urllib3==1.25.3
wrapt==1.11.2
```

**Comment by sfraint** [comment]
Fixed in #125, #127, which are now in [Netconan v0.10.2](https://pypi.org/project/netconan/0.10.2/)
